ruby/rails/security/brakeman/check-unsafe-reflection-methods.yaml

```diff
@@ -22,9 +22,11 @@ rules:
       - pattern-inside: |
           $X. ... .to_proc
       - pattern-inside: |
-          $Y.method(...)
+          $Y.method(<... $X ...>)
```

Comment:
Deep ellipsis is considered harmful by our engine team - it's supported but the performance is poor due to the recursive behavior under the hood.

Consider a `focus-metavariable` approach like the following: https://semgrep.dev/playground/s/pKLNe

Reply:
thanks! fixed!